charts/sn-platform/values.yaml

```diff
@@ -288,26 +290,33 @@ tls:
   broker:
     enabled: false
     cert_name: tls-broker
+    certSecretName:
   functions:
     enabled: false
     cert_name: tls-function
+    certSecretName:
   # settings for generating certs for bookies
   bookie:
     enabled: false
     cert_name: tls-bookie
+    certSecretName:
   # settings for generating certs for zookeeper
   zookeeper:
     enabled: false
     cert_name: tls-zookeeper
+    certSecretName:
   # settings for generating certs for recovery
   autorecovery:
     cert_name: tls-recovery
+    certSecretName:
   # settings for generating certs for toolset
   toolset:
     cert_name: tls-toolset
+    certSecretName:
```

Comment:
Shall we add some hints for users the difference between `cert_name` and `certSecretName` ?